.github/workflows/docker-images.yml

```diff
@@ -29,6 +29,8 @@ jobs:
       -
         name: Checkout
         uses: actions/checkout@v3
+        with:
```

Comment:
Why is this needed? This workflow is only used to *build* docker image for prepare Jekyll environments on our mirror server.